### PR TITLE
Remove outdated serializeUserData call in GLTFExporter

### PR DESCRIPTION
**Pull request by SBRK**

### CODE REVIEW

File: examples/js/exporters/GLTFExporter.js [hunk 1]
```diff
@@ -1922,7 +1922,7 @@ THREE.GLTFExporter.prototype = {
 
 			if ( scene.userData && Object.keys( scene.userData ).length > 0 ) {
 
-				gltfScene.extras = serializeUserData( scene );
+				gltfScene.extras = serializeUserData( scene, gltfScene );
```
Comment (donmccurdy) [inline examples/js/exporters/GLTFExporter.js]: Hm, I think something else is wrong here. Can this whole `if` block be removed? A few lines later on we have `serializeUserData( scene, gltfScene );` which should do everything that is needed. `serializeUserData` doesn't return anything.

Comment (SBRK) [reply examples/js/exporters/GLTFExporter.js]: Hmm, yeah I might have gone too fast on this one. I was getting a warning in the console in that particular call, and I didn't see the second call to serializeUserData with scene. Weird, I'm pretty sure I didn't have extras on my scene on the resulting GLTF. But again, I might have looked too quick...

The code I "fixed" must be leftover code from an old version.

Comment (donmccurdy) [reply examples/js/exporters/GLTFExporter.js]: Yeah, I think we must have missed this when refactoring the userData writing. Thanks!

**Review by donmccurdy — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**